src/costmap/src/Costmap.cpp

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "costmap/Costmap.hpp"
+#include <tf2/LinearMath/Quaternion.h>
+
+using std::vector;
+
+// TODO(orduno): Port ROS1 Costmap package
+Costmap::Costmap(rclcpp::Node* node) : node_(node)
+{
+  RCLCPP_INFO(node_->get_logger(), "Costmap::Costmap");
+}
+
+void
+Costmap::getCostmap(const nav2_msgs::msg::Costmap& specifications, nav2_msgs::msg::Costmap& costmap)
+{
+  RCLCPP_INFO(node_->get_logger(), "Costmap::getCostmap");
+
+  // TODO(orduno): faking out a costmap for now
+  (void)specifications;
```

Comment:
Instead of casting to void, you can also comment out the name ("specifications") in the function interface (line 27). I prefer commenting out the name as it results in fewer lines of code.

Reply:
Agree